### PR TITLE
Adding in addresses

### PR DESCRIPTION
**Pull request by olsona**
Adding in addresses, cleaning stuff up

### CODE REVIEW

File: portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactToVCardConverterTest.java [hunk 1]
```diff
@@ -151,6 +124,47 @@ public void testConversionToVCardNames() {
...
+
+    Person person = DEFAULT_PERSON.setAddresses(Arrays.asList(altAddress, primaryAddress));
+
+    // Run test
+    VCard vCard = GoogleContactToVCardConverter.convert(person);
+
+    // Check results for correct values and preferences
+    List<Address> actualPrimaryAddressList =
+        getPropertiesWithPreference(vCard, Address.class, VCARD_PRIMARY_PREF);
+    assertThat(actualPrimaryAddressList.size()).isEqualTo(1);
```
Comment (holachuy) [inline portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactToVCardConverterTest.java]: Can this be remove cause the next statement seems to verify there is only one element?

File: portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactToVCardConverterTest.java [hunk 2]
```diff
@@ -151,6 +124,47 @@ public void testConversionToVCardNames() {
...
+    assertThat(actualPrimaryAddressList.stream()
+        .map(Address::getStreetAddress).collect(Collectors.toList()))
+        .containsExactly(primaryStreet);
+    List<Address> actualAltAddressList =
+        getPropertiesWithPreference(vCard, Address.class, VCARD_PRIMARY_PREF + 1);
+    assertThat(actualAltAddressList.size()).isEqualTo(1);
```
Comment (holachuy) [inline portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactToVCardConverterTest.java]: Same here

File: portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactToVCardConverterTest.java (hunk 1 above)
Comment (olsona) [reply portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactToVCardConverterTest.java]: You are correct!

File: portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactToVCardConverterTest.java (hunk 2 above)
Comment (olsona) [reply portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactToVCardConverterTest.java]: Fixed.

**Pull request merged**